### PR TITLE
sdk: fix bug while activating the bundle plugin

### PR DESCRIPTION
**Pull request by xico42**
For some reason the bundle plugin activation with multiple optimized bundles, builded in v1 syntax, results in parsing error.

During the activation of the bundle, when we need to parse the rego modules, the ParserOptions is not forwarded to the `bundle.ActivateOps`, so the v1 syntax does not seem to be properly parsed.

### Why the changes in this PR are needed?

The OPA SDK does not initialize when provided multiple bundles that were optimized with v1 syntax.

### What are the changes in this PR?

Basically, I just forward the ParserOptions from the plugin manager to the `bundle.ActivateOpts` struct.

### Notes to assist PR review:

I am not totally sure about what the root problem is, but I was able to provide a failing test
and the fix.

It might be related to the issue #6450

### CODE REVIEW

**Review by johanfylling — COMMENTED** [review]
Great find @xico42 ! 😃 

This is unrelated to #6450.

I've pushed an updated test to your branch to further assert your fix.

Just a minor nit before we can get this merged.

File: sdk/opa_test.go [hunk 1]
```diff
@@ -2844,3 +2844,54 @@ func toMetricMap(metrics []*promdto.MetricFamily) map[string]bool {
 	}
 	return metricMap
 }
+
+func TestActivateOptimizedV1Bundles(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (xico42) [reply sdk/opa_test.go]: @johanfylling I've renamed the test and simplified the test structure to better reflect this.

[the remark this answers is not in the document]
Comment (johanfylling) [reply sdk/opa_test.go]: Thank you :)

**Review by johanfylling — APPROVED** [review]
Great work!
Thank you for your contribution! 😃
